tests/integration/project/cmd_project_test.go

```diff
@@ -56,6 +56,17 @@ var _ = Describe("odo project command tests", func() {
 		})
 	})
 
+	Context("odo machine readable output on get project", func() {
```

Comment:
I think we don't an extra context for this test. We can put this test under the context `Machine readable output tests` above

Reply:
updated, yes, that's reasonable change..